### ISSUE TITLE
Regression using 'yast' tool not displaying lines properly

### ISSUE THREAD

**Issue by igor2x** [mRemoteNG/mRemoteNG#460]
Hi,
I have talked to my colleague that is still using mRemoteNG 1.74 and asked him why hasn't he migrated to 1.75 like I have migrated and response was that he is frequently using "yast" tool on Suse Linux (I never use this tool) and in mRemoteNG v1.74 it properly displays lines, but in version 1.75 it looks regression in mRemoteNG not properly displaying lines.

How to reproduce:
1. Login to Suse Linux Enterprise Server using SSH2 protocol.
2. On bash shell type in: yast
   And problem appears. Instead of lines it is displayed letters.

Working fine in mRemoteNG 1.74.6023.15437
Problem in mRemoteNG 1.75 1.75.7000.19221

See print-screens (click on images to enlarge):
https://imgur.com/a/QAysO
https://imgur.com/a/2EqDN

Maybe there is some problem with displaying characters in second half of code page above 128 characters, because this lines characters are above 128 character. Don't know if this is just assumption.
Regards

**Comment by kmscode** [comment]
The version of PuTTYNG provided in both 1.74 and 1.75 is the same: 0.67 - however they were signed at different times. File size is identical, version is identical, timestamp is different due to signing procedures during the build process. This was corrected in commit 30b37951b26257cd81d76887ceb007bd763afac7

1.76 (or possibly a new 1.75 hotfix) will likely get a new PuTTYNG build of 0.68.

However, regardless of the PuTTYNG versions, this appears to be an environment issue and NOT an mRemoteNG/PuTTYNG issue:
http://www.novell.com/support/kb/doc.php?id=7015165

I've reproduced the character printing with BOTH 1.74 and 1.75 out of the box on a newly provisioned SLES 12 system.

```export NCURSES_NO_UTF8_ACS=1``` resolved by problem in both versions.

**Comment by igor2x** [comment]
Hi,
I try adding export NCURSES_NO_UTF8_ACS=1 to .bashrc and logout/login and this solution does not work for me.

It looks on this machine language is set to POSIX, executing 'locale' command.

I have read a lot of forums and this was most helpful:
https://serverfault.com/questions/475925/how-to-fix-putty-showing-garbled-characters

I don't want to mess with language settings on machines because everything else is working without a problem, the only exception is yast not displaying characters properly.

Temporally I have fixed this problem by adding bellow text to .bashrc file
`alias yast='LC_CTYPE=sl_SI.UTF-8 NCURSES_NO_UTF8_ACS=1 yast'`

When executing yast from shell it sets variables before starting yast.

It is still interesting that mRemoteNG 1.74 on the same Linux machine works without a problem, but mRemoteNG 1.75 does not. In my humble opinion it also got to be something in mRemoteNG that has changed from 1.74 to 1.75 related to this problem. It may not be Putty, it may be something in mRemoteNG or similar.

Is there anyway I can debug this problem in mRemoteNG?

I know it looks like something related to environment, but why does 1.74 works without a problem and 1.75 has problem displaying lines in yast?
The best variant in our case would still be to debug problem in mRemoteNG and fix the problem without changing Linux environments on this and many may many more machines.
Regards

**Comment by igor2x** [comment]
Hi,
now I have gone through all of the Putty setting at my colleague and compare them with my settings them with my settings and I have found out there was one setting in my colleague that was not related to mRemoteNG.

1. mRemoteNG: Tools | Options | Advanced | Launch PuTTy.
2. Window | Translation | Remote character set | by default setting is set "UTF-8", but at my colleague it was set to ISO-8859-2 (Latin-2).

The scenario was likely the following:
1. He installed 1.74 many years back and then he changed above setting to ISO-8859-2 and forgot about this setting.
2. Then he downloaded 1.75-portable with of course default value "UTF-8" tested "yast" program and it was not working. He assumed there is some regression problem in 1.75-portable.

We did the test:
1. Downloaded 1.75 again in Putty set to "ISO-8859-2" and yast is working fine, displaying line characters correctly.
2. Then we downloaded mRemoteNG 1.75 (non-portable) and installed it over existing mRemoteNG 1.74 and of course ISO-8859-2 was still present from 1.74 version and now yast is also correctly displaying characters.

In the end, we are not planning to change any setting on server environment. I asked why is the language set to POSIX and not to UTF-8 on Linux server and got response that there are some products that do not work correctly using UTF-8, so servers are set to POSIX instead.

It looks it was a setting problem. Sorry for false report.
Regards
